completers/git_completer/cmd/push.go

```diff
@@ -42,6 +42,8 @@ func init() {
 	pushCmd.Flags().BoolP("verbose", "v", false, "be more verbose")
 	rootCmd.AddCommand(pushCmd)
 
+	pushCmd.Flag("force-with-lease").NoOptDefVal = " "
```

Comment:
This is unrelated to the feature but was something I was using while testing this.